discord_bot/discord_db/db.py

```diff
@@ -1,19 +1,19 @@
-"""configure and create the database"""
-
-import os
-
-from dotenv import load_dotenv
-from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker
-
-from .modules import Base
-
-load_dotenv()
-
-DB_URL: str = os.getenv("PG_DATABASE_URL")
-
-engine = create_engine(DB_URL)
-
-Base.metadata.create_all(bind=engine)
-
-Session = sessionmaker(bind=engine)
+"""configure and create the database"""
+
+import os
+
+from dotenv import load_dotenv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from .modules import Base
+
+load_dotenv()
+
+DB_URL: str = os.getenv("DB_URL")
```

Comment:
os.getenv("DB_URL") 
add default input 
example if you cant get DB_URL get "SOME DB"